lisp/wayland/Makefile.am

```diff
@@ -16,21 +17,33 @@ GUILE_WARNINGS =                                          \
   -Wformat
 
 SUFFIXES = .scm .go
+
 .scm.go:
-	$(AM_V_GEN)$(top_builddir)/pre-inst-env $(GUILE_TOOLS) compile $(GUILE_TARGET) $(GUILE_WARNINGS) -o "$@" "$<"
+	$(AM_V_GEN)$(top_builddir)/pre-inst-env $(GUILE_TOOLS) compile $(GUILE_TARGET)
+
+$(GUILE_WARNINGS) -o "$@" "$<"
```

Comment:
makefile's rule looks likes 
``` makefile
target: dependencies
   command
   ...
```

see https://web.mit.edu/gnu/doc/html/make_4.html

so 

``` makefile
.scm.go:
	$(AM_V_GEN)$(top_builddir)/pre-inst-env $(GUILE_TOOLS) compile $(GUILE_TARGET) \
        $(GUILE_WARNINGS) -o "$@" "$<"
```

Reply:
Thanks for really fast replies. I need guile wayland bindings (not all of them just a little part) for my personal project and it's the only suitable repo I found that still requires some DIY. 

I am very thankful for all the work you've done. All the formatting is totally up to my taste, I'm not judging the formatting of yours in any way. And I'm just started making the changed I want.  

Besides It's my first time into Make and Autotools.

Reply:
I don't even insist on the PR being merged. Will convert to draft.

Reply:
> Thanks for really fast replies. I need guile wayland bindings (not all of them just a little part) for my personal project and it's the only suitable repo I found that still requires some DIY. 

if you need any miss wayland ffi binding, you can open a issue/PR, i will add it. 

> All the formatting is totally up to my taste, I'm not judging the formatting of yours in any way. And I'm just started making the changed I want.

This is not formatting issues, it's makefile's syntax error, https://github.com/guile-wayland/guile-wayland/blob/1ec967d427e67310135248d9a916735d595230ae/lisp/wayland/Makefile.am#L22-L23 fix it, and make `build` phase work.